cmd/influx/bucket_schema.go

```diff
@@ -36,8 +36,9 @@ func getBucketSchemaClient(ctx *cli.Context) bucket_schema.Client {
 
 func newBucketSchemaCmd() cli.Command {
 	return cli.Command{
-		Name:  "bucket-schema",
-		Usage: "Bucket schema management commands",
+		Name:   "bucket-schema",
+		Usage:  "Bucket schema management commands",
+		Before: middleware.NoArgs,
```

Comment:
NoArgs is a weird name for this - I would call it something like `SubcommandValid`

Reply:
The name & error message mirror the `cobra` equivalent. It's meant to say "this command doesn't take any positional args, so if you see one try parsing it as a subcommand" which then leads to "unknown command" errors... agreed it's not always the most ideal way to think about it.